server/testutil/testenv/testenv.go

```diff
@@ -65,6 +65,9 @@ executor:
   # If we don't actually have the memory, we'll OOM, which is OK
   # for testing purposes.
   memory_bytes: 10_000_000_000
+  # Make sure Bazel can clean up the sandbox when running docker tests locally.
+  # Without this, containers run as root, so outputs can't be cleaned up.
+  docker_inherit_user_ids: true
```

Comment:
@luluz66 @bduffany do we need to/will we support this in podman too?

Reply:
It looks like podman inherits user IDs by default, so we probably don't need to add an explicit option for it:

```shell
mkdir -p /tmp/foo
podman run -v /tmp/foo:/tmp/foo ubuntu:20.04 touch /tmp/foo/bar.txt
stat /tmp/foo/bar.txt
# Output: Access: (0644/-rw-r--r--)  Uid: ( 1000/       b)   Gid: ( 1000/       b)
```

vs docker

```shell
rm -rf /tmp/foo
mkdir -p /tmp/foo
docker run -v /tmp/foo:/tmp/foo ubuntu:20.04 touch /tmp/foo/bar.txt
stat /tmp/foo/bar.txt
# Output: Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)

```

Reply:
do we still need this here?

Reply:
not really -- removed